srcpkgs/ol/template

```diff
@@ -10,9 +10,13 @@ license="LGPL-3.0-or-later, MIT"
 homepage="https://yuriy-chumak.github.io/ol/"
 changelog="https://raw.githubusercontent.com/yuriy-chumak/ol/master/doc/CHANGELOG.md"
 distfiles="https://github.com/yuriy-chumak/ol/archive/${version}.tar.gz"
-checksum=1f3da2d3b2e47f5c626bfe2727b649db3fd8f53e700dad2a984314275ee7a6ff
+checksum=5a7b8ff5eeba2db63ba8d409c6ebc3af5f9cb21d14a2ec1a279d8137298360ef
 
-CFLAGS="-lm"
+case "$XBPS_TARGET_MACHINE" in
+	i686*|x86_64*) makedepends+=" gcc-multilib";;
+esac
+
+CFLAGS="${CFLAGS/-D_FORTIFY_SOURCE=2/} -lm -DOLVM_FFI -DHAS_SOCKETS -DHAS_DLOPEN"
```

Comment:
why remove FORTIFY_SOURCE?

Reply:
1. Ol tested and works for years without additional impact of "_FORTIFY_SOURCE",
1. "_FORTIFY_SOURCE" checks for Ol is mainly useless due to:
   1. no `printf` usage,
   2. no system memory allocations - Ol has it's own heap with GC (only one `malloc` and `realloc`s if necessary)
   3. no `memmove`, `bcopy`, etc.
1. Annoying multiple warnings `#  warning _FORTIFY_SOURCE requires compiling with optimization (-O)` in `make check` (that makes checks for 32- and 64-, debug, release, mingw targets).

--
In fact, a separate test cycle for "__USE_FORTIFY_LEVEL" is planned. But at the moment no such a test target is prepared.

Reply:
> => ol-2.3.6_1: running do_check ...
> Building test binaries:

does that mean that the installed binaries are not tested, and test run on different ones?

Reply:
Good question. No and yes.

The test binaries builds separately using separate makefile. But one of built test binary is the same as installed binary in sense "same compiler, same command line, same options -> same binary".